components/chip/overview.md

```diff
@@ -66,6 +66,8 @@ The table below lists the Chip parameters. Also check the [Chip API Reference](/
 | `Icon` | `object` | Use this parameter to render an icon in the chip. |
 | `Removable` | `bool` | Specifies if the chip can be removed by the user. If set to `true` a remove icon will be rendered on each available chip. |
 | `RemoveIcon` | `object` | Defines the icon that will be rendered if the `Removable` parameter is set to `true`. |
+| `Selectable` | `bool` <br/> (`true`) | Specifies if the Chip can be selected. Overrides the `Selected` parameter value.|
```

Comment:
```suggestion
| `Selectable` | `bool` <br/> (`true`) | Specifies if the Chip can be selected. If set to `false`, it will override the `Selected` parameter value.|
```